src/axom/sidre/examples/spio/CMakeLists.txt

```diff
@@ -9,16 +9,16 @@
 #------------------------------------------------------------------------------
 # List of single source file examples
 #------------------------------------------------------------------------------
-set(example_sources
+set(spio_example_sources
     IORead.cpp
     IOWrite.cpp
     )
 
 if (SCR_FOUND)
-    list(APPEND example_sources IO_SCR_Read.cpp)
-    list(APPEND example_sources IO_SCR_Write.cpp)
-    list(APPEND example_sources IO_SCR_Checkpoint.cpp)
-    list(APPEND example_sources IO_SCR_Output.cpp)
+    # TODO: Add these as a headers to a unified example driver and can be run as tests?
```

Comment:
I think this is something for me to look at now that SCR is re-enabled.  They might be redundant with  SCR_Checkpoint, but I need to give it more thought.